qemu/tests/cfg/slof_multi_devices.cfg

```diff
@@ -0,0 +1,28 @@
+- slof_multi_devices: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = slof_multi_devices
+    restart_vm = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
+    start_vm = no
+    pci_controllers = ''
+    only ppc64le ppc64
+    only Linux
+    # select one drive format since just test slof
+    # with pci-bridge and virtio-net-pci.
+    only virtio_blk
+    variants:
+        - with_pci_bridge:
+            device_type = 'pci-bridge'
+            pci_controllers +=" pci_bridge0"
```

Comment:
It does not need to assign value to "pci_controllers" and "type_pci_bridge0" here, it's better add a parameter that we could set it as our need to define the number of pci-bridge.